spec/spec_helper.rb

```diff
@@ -276,7 +276,6 @@ def node_files
     profile::core::nfsclient
     profile::core::nfsserver
     profile::core::sysctl::lhn
-    python
```

Comment:
As noted elsewhere, there needs to be replacement spec coverage for the installation of python.